kratos/tests/sources/test_nodal_data.cpp

```diff
@@ -80,12 +80,10 @@ namespace Kratos {
 					velocity_y_sum += i_node->FastGetSolutionStepValue(VELOCITY_Y);
 				}
 				KRATOS_CHECK_DOUBLE_EQUAL(pressure_sum, i * size);
-				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, i * size);
+				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, 2*i * size);
 				KRATOS_CHECK_DOUBLE_EQUAL(velocity_y_sum, 0.00);
 			}
 		}
```

Comment:
other iteration. 

I guess it might happen also with the old version, in the case one defines for example the variable VELOCITY_XA, this might enter in the map between VELOCITY_X and VELOCITY_Y.

bad thing is  that i believe this would lead to a buggy behaviour, or as a very minimum to a bad performance. @pooyan-dadvand am i wrong about this?